Makefile

```diff
@@ -29,7 +29,7 @@ test: deps
 
 # build will ensure all of our tests pass and then build the go binary
 build: test
-	go build -mod=vendor ./...
+	GOPROXY="https://proxy.golang.org" go build -mod=vendor ./...
 
```

Comment:
Do we have to redefine `GOPROXY` even when it was already set in the Dockerfile?

Reply:
I tried the following and it also seemed to have worked:
```
╰─➤  $GOPROXY go build .
```
I'm not saying we should change it I'm just wondering why this specific syntax?

Reply:
Sure, let's use `$GOPROXY` instead :)
As to why redefining it in Makefile and in Dockerfileis just because not sure everyone use Docker directly to launch (I don't locally for example). I can remove it if you want though.